docs/meshstack.mesh-policies.md

```diff
@@ -0,0 +1,38 @@
+---
+id: meshstack.mesh-policies
+title: Mesh Policies
+---
+
+meshPolicies can be set up between meshObjects, i.e., meshCustomer, meshProject, meshLandingZone and
```

Comment:
I think it will help to describe that a policy can have multiple rules and that every rules has to apply in order for a policy to apply. But only 1 policy needs to match in order to assign the two objects. https://docs.google.com/presentation/d/1ceXtt6ITMOXMhjPcbAb0jvJyHSk_fHaDLPzSQJd1Qak/edit#slide=id.g913724683f_0_415 slide 24 and 25 make a visual example we could incorporate

Reply:
I have added it to the Working explanation section. Please check it.